frontend/src/App.css

```diff
@@ -62,9 +62,9 @@ body {
 }
 
 .main {
-    background-image: url(assets/backgrounds/App-root-bg.svg);
+    background-image: url(assets/backgrounds/background_lightblue.svg);
```

Comment:
Why not just completely overwrite the old background svg instead?

Reply:
Oh, you're right. Doing that now.